build-system/tasks/storybook-react/index.js

```diff
@@ -0,0 +1,105 @@
+('use strict');
+
+const argv = require('minimist')(process.argv.slice(2));
+const path = require('path');
+const {build: buildBinary} = require('../build');
+const {createCtrlcHandler} = require('../../common/ctrlcHandler');
+const {cyan} = require('kleur/colors');
+const {defaultTask: runAmpDevBuildServer} = require('../default-task');
+const {exec, execScriptAsync} = require('../../common/exec');
+const {isCiBuild} = require('../../common/ci');
+const {log} = require('../../common/logging');
+
+/** @type {Object<StorybookEnv, number>} */
+const ENV_PORTS = {
+  amp: 9001,
+  preact: 9002,
+  react: 9003,
+};
+
+const repoDir = path.join(__dirname, '../../..');
+
+/** @typedef {'amp'|'preact'|'react'} StorybookEnv */
+
+/**
+ * @param {StorybookEnv} env
+ * @return {string}
+ */
+const envConfigDir = (env) => path.join(__dirname, `${env}-env`);
+
+/**
+ * @param {StorybookEnv} env
+ */
+function launchEnv(env) {
+  log(`Launching storybook for the ${cyan(env)} environment...`);
+  const {'storybook_port': port = ENV_PORTS[env]} = argv;
+  execScriptAsync(
+    [
+      './node_modules/.bin/start-storybook',
+      `--config-dir ${envConfigDir(env)}`,
+      `--static-dir ${repoDir}/`,
+      `--port ${port}`,
+      '--quiet',
+      isCiBuild() ? '--ci' : '',
+    ].join(' '),
+    {cwd: __dirname, stdio: 'inherit'}
+  ).on('error', (err) => {
+    throw new Error('Launch failed', {cause: err});
```

Comment:
Looks like typecheck doesn't like `{cause: err}`. I'd say it's ok to remove `cause`.

https://app.circleci.com/pipelines/github/ampproject/amphtml/21594/workflows/f5a66572-3edb-4cd2-a7bb-5f18c4054ed3/jobs/421912?invite=true#step-116-211